src/scip.rs

```diff
@@ -46,6 +46,12 @@ impl ScipPtr {
         Ok(())
     }
 
+    pub(crate) fn set_bool_param(&mut self, param: &str, value: u32) -> Result<(), Retcode> {
```

Comment:
It's probably better to make `value` a `bool` here and transform it to an `u32` in this function instead of the other one. That way, this `set_bool_param` can be called correctly from other places.